Fixed the timestamp error

sreejakshetty opened the pull request:
#78

sreejakshetty committed:
Register Page View

DishantK1807 committed:
Updated for User and Admin Settings page

sreejakshetty committed:
Made the background white for clear view

sreejakshetty committed:
Moved the required fields to top of the page

sreejakshetty committed:
Fixed different timestamps error

sreejakshetty committed:
Added start time and end time to shifts table

sreejakshetty committed:
Added fields start_time and end_time to shifts

sreejakshetty committed:
Fixed the timestamp error for log and delete

sreejakshetty committed:
added start_time and end_time

kamishettysreeja25 commented:
![screenshot from 2016-06-10 19 34 07](https://cloud.githubusercontent.com/assets/14835922/15966804/5672f0ba-2f42-11e6-9c01-02ce990c72c8.png)
![screenshot from 2016-06-10 19 33 50](https://cloud.githubusercontent.com/assets/14835922/15966806/56ca51d4-2f42-11e6-9a99-7d702f54f748.png)
![screenshot from 2016-06-10 19 33 35](https://cloud.githubusercontent.com/assets/14835922/15966808/56d19f34-2f42-11e6-83de-d7177cace6ff.png)
![screenshot from 2016-06-10 19 33 09](https://cloud.githubusercontent.com/assets/14835922/15966807/56d14868-2f42-11e6-84b4-6f703bb6ab17.png)
![screenshot from 2016-06-10 19 32 54](https://cloud.githubusercontent.com/assets/14835922/15966809/56d4335c-2f42-11e6-863f-f429443dbc97.png)

kamishettysreeja25 commented:
test on online server

DishantK1807 commented:
@kamishettysreeja25 : I see here that you changed install.sql file here. Please make those changes in update.sql file as we are not changing the install.sql file.

kamishettysreeja25 commented:
@DishantK1807 sure. I didn't know about that. I will add the fields in update.sql and undo them in install.sql.
Thanks

sreejakshetty committed:
undo the changes made for start and end time

sreejakshetty committed:
Added fields start_time and end_time to update.sql

kamishettysreeja25 commented:
I have undo the changes made in install.sql and added table entries in update.sql.Please review the PR

hongquan commented:
@kamishettysreeja25 I recently merged some PR and make this PR have conflict. Can you please merge from master to this branch to fix the conflict?

sreejakshetty committed:
Merge branch 'master' into error_shifts

Conflicts:
	db/update.sql

sreejakshetty committed:
added start_time and end_time fields

kamishettysreeja25 commented:
@hongquan  I have fixed the conflicts . Can you review it and merge it. 
Thanks :)

DishantK1807 committed:
Updated user_model.php 

updated for the added items in the "user's" table
(current_city, facebook, twitter, github, organization, organization_web)

DishantK1807 committed:
Update User_model.php

DishantK1807 committed:
Update user_settings.php

hongquan committed:
Merge pull request #71 from fossasia/admin_setings

Updated for User and Admin Settings page

hongquan commented on includes/pages/user_shifts.php:
```diff
@@ -39,7 +42,7 @@ function user_shifts() {
         engelsystem_error('Unable to delete shift entry.');
 
-      engelsystem_log("Deleted " . User_Nick_render($shift_entry_source) . "'s shift: " . $shift_entry_source['name'] . " at " . $shift_entry_source['Name'] . " from " . date("Y-m-d H:i", $shift_entry_source['start']) . " to " . date("Y-m-d H:i", $shift_entry_source['end']) . " as " . $shift_entry_source['angel_type']);
+      engelsystem_log("Deleted " . User_Nick_render($shift_entry_source) . "'shift: " . $shift_entry_source['name'] . " at " . $shift_entry_source['Name'] . " from " . date("Y-m-d", $shift_entry_source['start']) . " " . date("H:i", $shift_entry_source['start_time']) . " to " . date("Y-m-d", $shift_entry_source['end']) . " " . date("H:i", $shift_entry_source['end_time'])  . " as " . $shift_entry_s ...
```
Why is there change from "someone's shift" to "someone'shift"?

kamishettysreeja25 commented:
@hongquan 
I have mistakenly changed it . I will change it to " someone's shift ".
Thank you

sreejakshetty committed:
changed from someone shift to someone's shift

hongquan commented:
Please clean trailing whitespaces (I reminded this in Slack chat)

hongquan committed:
Merge pull request #93 from fossasia/user_model

Add fields to User model.

Close #88

sreejakshetty committed:
Indented the code and removed trailing spaces

kamishettysreeja25 commented:
@hongquan I have indented the code and removed the trailing spaces . 
Thanks.

hongquan commented:
@kamishettysreeja25 Please read again: https://www.drupal.org/coding-standards#functcall

- There should be no space around the brackets

```
$var = foo($bar, $baz, $quux);
```

- The indentation width is 2 spaces, not 8 spaces (https://www.drupal.org/coding-standards#indenting). The original code follows this well.
- There should be one blank line at the end of file (you can configure editor to add it automatically)

kamishettysreeja25 commented:
@hongquan Sure . I will follow this from now. I have pushed the code before I read the coding conventions given by you.I will make necessary changes and push the code.
Cheers

sreejakshetty committed:
added standard indentation

kamishettysreeja25 commented:
@hongquan I have changed the files . Added 2 space instead of 8 spaces and removed extra spaces at end of brackets " ) " and added one blank line at the end of file.

hongquan commented:
Why do you change the braces from

```php
function do() {
  // blah blah
}
```
to

```php

function do()
{
  // blah blah
}
```

The original code is already compliant with Drupal convention (https://www.drupal.org/coding-standards#functdecl).

sreejakshetty committed:
changed the functions indentation

kamishettysreeja25 commented:
@hongquan I have used a php formatter and it created the indentation from 
```
fun(){ 
// code
}
```
to 
```fun
fun(){
//code
}
```
now , I have changed it .
Thank you

hpdang committed:
Merge pull request #69 from fossasia/UI/UX

Ui/UX Register Page View

hongquan commented:
There still be a lot of trailing spaces. Please look into https://github.com/fossasia/engelsystem/pull/87/files to check.

There are some blank lines to separate block of codes, to make it easier to read. Don't delete them (but still clean the trailing spaces on them).

Some lines have wrong indentation.

sreejakshetty committed:
Merge remote-tracking branch 'origin/error_shifts' into error_shifts

sreejakshetty committed:
timestamp error,map view and code standards

hongquan commented on includes/pages/admin_shifts.php:
```diff
@@ -195,22 +191,22 @@ function admin_shifts() {
-              'start_time' => $start_time,
-              'end_time'  => $end_time,
+              'start_time' => DateTime::createFromFormat("H:i", date("H:i",$shift_start))->getTimestamp(),
+              'end_time'  =>DateTime::createFromFormat("H:i", date("H:i",$shift_end))->getTimestamp(),
```
You lack space after `=>`

sreejakshetty committed:
Fixed time stamp error

hongquan commented on includes/pages/admin_shifts.php:
```diff
@@ -195,22 +191,22 @@ function admin_shifts() {
           
           $shift_start = $shift_end;
-        } while ($shift_end < $end);
+        } while ($shift_end < $end );
```
Why the space before `)`?

hongquan commented on includes/pages/user_settings.php:
```diff
@@ -5,7 +5,7 @@ function settings_title() {
 
 function user_settings() {
-  global $enable_tshirt_size, $tshirt_sizes, $themes, $locales;
+  global $enable_tshirt_size, $tshirt_sizes, $themes, $locales ;
```
why space between `$locales` and `;`?

hongquan commented on includes/pages/user_settings.php:
```diff
@@ -200,9 +206,61 @@ function user_settings() {
+}
+
+// User settings page
+if( $_SESSION['uid'] > 1){  
```
Please fix the space

hongquan commented on includes/pages/user_shifts.php:
```diff
@@ -4,6 +4,10 @@ function shifts_title() {
   return _("Shifts");
 }
 
+function edit_shifts_title(){
```
Please fix whitespace

hongquan commented on includes/pages/user_shifts.php:
```diff
@@ -294,7 +316,7 @@ function user_shifts() {
         sql_query("INSERT INTO `UserAngelTypes` (`user_id`, `angeltype_id`) VALUES ('" . sql_escape($user_id) . "', '" . sql_escape($selected_type_id) . "')");
 
       $user_source = User($user_id);
```
Why there is difference in indentation between the line

```
sql_query("INSERT INT
```

and

```
$user_source = 
```

kamishettysreeja25 commented:
#87 #99 #91 all these issues are fixed
Tested on online server http://107.170.88.225/engelsystem/public/
![screenshot from 2016-06-15 07 47 07](https://cloud.githubusercontent.com/assets/14835922/16066009/5f1ad258-32cd-11e6-8b6a-9ac786296efc.png)

sreejakshetty committed:
removed trailing spaces

sreejakshetty committed:
removed debugging statements

kamishettysreeja25 commented:
@hongquan Removed all trailing spaces as mentioned .

hongquan commented:
@kamishettysreeja25 Don't fix only on the line I commented. You should be proactive. There are too many issues that I cannot comment all. The issue which I commented is repeated in other parts of file and other files in your commits. I told you to look into https://github.com/fossasia/engelsystem/pull/87/files to check on all your files.

kamishettysreeja25 commented:
@hongquan This PR even contains files which are already pushed into the master like user_settings.php and guest_login.php . The files I changed are admin_shifts.php , user_shifts.php , Shifts_model.php,update.sql.I would like to  create a new PR as this PR contains the merge commits and others files also .
Thank you.

kamishettysreeja25 commented:
Fixed #106 here

hongquan commented:
Did you **really** go to the link? The link only list the changes made by you. I only tell you to check the change **made by you**. Those already in master is even not shown there.

![selection_094](https://cloud.githubusercontent.com/assets/314607/16068851/b6cd30a6-32f3-11e6-8eae-859e4a54d4f2.png)

kamishettysreeja25 commented:
@hongquan I was talking about the commits which were merged into master and which were present in this branch. I am not blaming master .
![screenshot from 2016-06-15 11 23 16](https://cloud.githubusercontent.com/assets/14835922/16069407/d4dfa31a-32eb-11e6-97f8-9d2e09d51eae.png)
I have gone through the link went through the code and I have made necessary changes to the files and create a new PR #106 , as this PR contains the files which were already merged and has merge  commits.
![screenshot from 2016-06-15 11 17 13](https://cloud.githubusercontent.com/assets/14835922/16069507/ac11b2e2-32ec-11e6-9e10-e9d2f9c6d76e.png)
![screenshot from 2016-06-15 11 16 25](https://cloud.githubusercontent.com/assets/14835922/16069508/ac16c9bc-32ec-11e6-8ae9-f945699e2e62.png)
Cheers and Thank you :)

hongquan commented:
What is the purpose of #106? It is to replace this PR and this PR should be closed (without merging)?
- If yes, why do you have to create a new PR just to do the same thing as this PR?
- If no, what is the difference between this PR and #106?

The links https://github.com/fossasia/engelsystem/pull/87/files show the difference between your branch and master. That difference is caused by your change of code in your branch. Any files which is in master and is not different between your branch and master will not be shown in that links.

The change that I showed you in the picture above is the change made by this PR. That change (and other similar) is not qualified and this PR cannot be merged. If you refuse to fix, this PR can only be closed.

hongquan commented:
> this PR contains the files which were already merged

This argument is irrelevant, because the difference on the file, between your branch and master will be zero and it won't be shown here for me to review.

kamishettysreeja25 commented:
>What is the purpose of #106? It is to replace this PR and this PR should be closed (without merging)?
>
> -    If yes, why do you have to create a new PR just to do the same thing as this PR?
> -    If no, what is the difference between this PR and #106?

#106 takes care of both PR #87  #83  .These both PR's can be closed.
![screenshot from 2016-06-15 13 20 16](https://cloud.githubusercontent.com/assets/14835922/16072193/40933656-32fd-11e6-96dc-83cb676c5a1e.png)
As I see in the link https://github.com/fossasia/engelsystem/pull/87/files . I see 8 files being changed .
I taught there is a conflict in my branch as I fetched the code from master and pushed it to error_shifts branch.
So I created a new PR to tackle this issue. I have fixed all the issues in #106 .
